tests/integration/test_api.py

```diff
@@ -655,6 +655,26 @@ def test_get_unanswered_questions_api(client):
     assert data["status"] == "found"
 
 
+def test_get_unanswered_questions_api_with_country_filter(client):
+    ProductInsight.delete().execute()  # remove default sample
+
+    # test for filter with "country"
+
+    d = ProductInsightFactory(
```

Comment:
You don't have to assign the product to `d` variable as you're not using it (local variable 'd' is assigned to but never used)